Library/Homebrew/cmd/install.rb

```diff
@@ -52,7 +52,10 @@
 #:    are not deleted.
 #:
 #:    If `--build-bottle` is passed, prepare the formula for eventual bottling
-#:    during installation.
+#:    during installation. A formula installed this way is not guaranteed to be
```

Comment:
```suggestion
#:    during installation. A formula installed this way is not meant to be
```

I think specifying the intention instead of the effect here makes it clearer to the user.